pygsti/tools/fogitools.py

```diff
@@ -865,190 +830,6 @@ def _create_errgen_op(vec, list_of_mxs):
     return ret
 
 
-#An alternative but inferior algorithm for constructing FOGI quantities: Keep around for checking/reference or REMOVE?
```

Comment:
Same as above, still under active development so I am not ready to jettison this as flotsam yet.